pkg/report/report.go

```diff
@@ -273,20 +278,15 @@ var dynamicTitleReplacement = []replacement{
 		regexp.MustCompile(`([^a-zA-Z0])(?:0x)?[0-9a-f]{6,}`),
 		"${1}ADDR",
 	},
-	{
-		// Replace that everything looks like a decimal number with "NUM".
-		regexp.MustCompile(`([^a-zA-Z0-9])[0-9]{5,}`),
-		"${1}NUM",
-	},
 	{
 		// Replace IP addresses.
 		regexp.MustCompile(`([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})`),
 		"IP",
 	},
 	{
 		// Replace that everything looks like a file line number with "LINE".
-		regexp.MustCompile(`(:[0-9]+)+`),
-		":LINE",
+		regexp.MustCompile(`(\.\w+)(:[0-9]+)+`),
```

Comment:
Maybe we should drop this ":LINE" replacement, what do you think?
"NUM" looks fine even for line numbers, and we don't even know if it's really a line number or not.

Reply:
In that case we'd have to update even more bugs, even on Linux :) 

Right now it seems that this regexp makes reasonable guesses. There were many false positives on gVisor and FreeBSD, but after the modification from this pull request they should be gone.

Reply:
I see only 16 LINE's here: https://syzkaller.appspot.com/upstream

Changed titles were a problem before we decided to update the database. But if we update it, it should not be an issue.